### ISSUE TITLE
Error during setup of component var with HA 0.108

### ISSUE THREAD

**Issue by gary-reyes** [snarky-snark/home-assistant-variables#26]
The component is unable to be loaded at startup when using HA 0.108 beta.

```
Error during setup of component var
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/setup.py", line 171, in _async_setup_component
    hass, processed_config
  File "/config/custom_components/var/__init__.py", line 105, in async_setup
    await component.async_setup(config)
  File "/usr/src/homeassistant/homeassistant/helpers/entity_component.py", line 131, in async_setup
    await asyncio.gather(*tasks)
  File "/usr/src/homeassistant/homeassistant/helpers/entity_component.py", line 229, in async_setup_platform
    self.hass, self.config, self.domain, platform_type
  File "/usr/src/homeassistant/homeassistant/setup.py", line 230, in async_prepare_setup_platform
    integration = await loader.async_get_integration(hass, platform_name)
  File "/usr/src/homeassistant/homeassistant/loader.py", line 323, in async_get_integration
    Integration.resolve_from_root, hass, components, domain
  File "/usr/local/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/usr/src/homeassistant/homeassistant/loader.py", line 156, in resolve_from_root
    manifest_path = pathlib.Path(base) / domain / "manifest.json"
  File "/usr/local/lib/python3.7/pathlib.py", line 920, in __truediv__
    return self._make_child((key,))
  File "/usr/local/lib/python3.7/pathlib.py", line 699, in _make_child
    drv, root, parts = self._parse_args(args)
  File "/usr/local/lib/python3.7/pathlib.py", line 653, in _parse_args
    a = os.fspath(a)
TypeError: expected str, bytes or os.PathLike object, not NoneType
```

**Comment by snarky-snark** [comment]
See #6.

**Comment by snarky-snark** [comment]
@gary-reyes, this should be fixed in 0.9.3
